cmake/app/boilerplate.cmake

```diff
@@ -74,11 +74,11 @@ add_custom_target(code_data_relocation_target)
 # when trying to reference ENV${ZEPHYR_BASE}.
 
 # Note any later project() resets PROJECT_SOURCE_DIR
-file(TO_CMAKE_PATH "$ENV{ZEPHYR_BASE}" PROJECT_SOURCE_DIR)
+file(TO_CMAKE_PATH "${ZEPHYR_BASE}" PROJECT_SOURCE_DIR)
 
 set(ZEPHYR_BINARY_DIR ${PROJECT_BINARY_DIR})
 set(ZEPHYR_BASE ${PROJECT_SOURCE_DIR})
-set(ENV{ZEPHYR_BASE}   ${ZEPHYR_BASE})
+set({ZEPHYR_BASE}   ${ZEPHYR_BASE})
```

Comment:
Shouldn't this line just be removed?

Reply:
seems like my sed has been playing tricks with me.
Thanks for spotting, this will be fixed.

Reply:
Fixed